tests/php/Forms/TextareaFieldTest.php

```diff
@@ -50,6 +51,30 @@ public function testReadonlyDisplaySpecialHTML()
         );
     }
 
+    /**
+     * Tests the TextareaField Max Length Validation Failure
+     */
+    public function testMaxLengthValidationFail()
+    {
+        $field = new TextareaField("Test", "Test");
+        $field->setMaxLength(5);
+        $field->setValue("John Doe"); // 8 characters, so should fail
+        $result = $field->validate(new RequiredFields());
```

Comment:
Do we need to pass in the `RequiredFields` validator? IIRC there was a change recently that made that not needed? @GuySartorelli do you know?

Reply:
A validator does have to be passed in. There is a new, simpler `FieldsValidator` that could be used instead but for the purposes of this test I don't think it really matters.